trail/query/templates/query.html

```diff
@@ -1,19 +1,54 @@
 {% extends "base.html" %}
+{% load static %}
+
 
 {% block query-nav-state%}active{% endblock %}
 
+
+{% block head.extend %}
+<style>
+ table, th, td {
+     border: 1px solid black;
+ }
+</style>
+{% endblock %}
+
+
 {% block content %}
 <!-- Post -->
 <section class="post">
 		<header class="major">
 				<h1>This is where <br />
 						the query lives</h1>
-				<p>Aenean ornare velit lacus varius enim ullamcorper proin aliquam<br />
-						facilisis ante sed etiam magna interdum congue. Lorem ipsum dolor<br />
-						amet nullam sed etiam veroeros.</p>
+				<p>The placeholder is removed here.<br />
+						Here as well.<br />
+						So as here.</p>
 		</header>
-		<div class="image main"><img src="images/pic01.jpg" alt="" /></div>
-		<p>Donec eget ex magna. Interdum et malesuada fames ac ante ipsum primis in faucibus. Pellentesque venenatis dolor imperdiet dolor mattis sagittis. Praesent rutrum sem diam, vitae egestas enim auctor sit amet. Pellentesque leo mauris, consectetur id ipsum sit amet, fergiat. Pellentesque in mi eu massa lacinia malesuada et a elit. Donec urna ex, lacinia in purus ac, pretium pulvinar mauris. Nunc lorem mauris, fringilla in aliquam at, euismod in lectus. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Curabitur sapien risus, commodo eget turpis at, elementum convallis enim turpis, lorem ipsum dolor sit amet nullam.</p>
-		<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis dapibus rutrum facilisis. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Etiam tristique libero eu nibh porttitor fermentum. Nullam venenatis erat id vehicula viverra. Nunc ultrices eros ut ultricies condimentum. Mauris risus lacus, blandit sit amet venenatis non, bibendum vitae dolor. Nunc lorem mauris, fringilla in aliquam at, euismod in lectus. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. In non lorem sit amet elit placerat maximus. Pellentesque aliquam maximus risus. Donec eget ex magna. Interdum et malesuada fames ac ante ipsum primis in faucibus. Pellentesque venenatis dolor imperdiet dolor mattis sagittis. Praesent rutrum sem diam, vitae egestas enim auctor sit amet. Pellentesque leo mauris, consectetur id ipsum.</p>
 </section>
+
+<form action="results" method="post">
+    {% csrf_token %}
+    {{ queryform }}
+    <input type="submit" class="btn-primaryg" value="submit" >
+</form>
+
+{% if render_table %}
+<h2>Results</h2>
+<table style="width:100%">
+    <tr>
+        <th>Instrument</th>
+        <th>Telescopename</th>
+        <th>datetime_begin</th>
+        <th>exposureduration</th>
```

Comment:
Please format these three table headers in a more human-readable friendly way. I suggest "Telescope," "Observation Start," and "Duration (s)".

Reply:
Fixed.